apps/aragon-fundraising/app/src/components/NewOrder/Order.js

```diff
@@ -104,9 +104,17 @@ const Order = ({ isBuyOrder }) => {
     return collateralItems[selectedCollateral].reserveRatio
   }
 
+  const getUserBalance = () => {
+    const balance = isBuyOrder ? [userDaiBalance, userAntBalance][selectedCollateral] : userBondedBalance
+    return fromDecimals(balance, bondedDecimals).toFixed(4)
```

Comment:
Not sure we need 4 decimals precision here. @osarrouy what do you think ?
(This shows the user balance within the New Order panel)

Reply:
@osarrouy sent a picture of how they showed balance in Uniswap and they are using 4 decimal precision.

Reply:
Also, prefer the `formatBigNumber` util